deployments/vanilla/terraform/min-iam-policy.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
is this updated? or should we remove it for now. I think we should find a way to plugin to test and then add it otherwise people will see failures

Reply:
will remove for now

Reply:
done